src/plugins/proxy/tests/CMakeLists.txt

```diff
@@ -7,7 +7,6 @@ set(TARGET_NAME ov_proxy_plugin_tests)
 set(DEPENDENCIES
     mock_engine
     openvino::runtime
```

Comment:
```suggestion
```
because only plugins need to be set as `DEPENDENCIES`

Reply:
Done, thanks!